client/my-sites/stats/hooks/use-stats-navigation-history.ts

```diff
@@ -28,6 +35,13 @@ const possibleBackLinks: { [ key: string ]: string | null } = {
 	subscribers: '/stats/subscribers/{period}/',
 	posts: '/stats/{period}/posts/',
 	authors: '/stats/{period}/authors/',
+	filedownloads: '/stats/{period}/filedownloads/',
+	referrers: '/stats/{period}/referrers/',
+	locations: '/stats/{period}/locations/',
```

Comment:
Should the legacy `countryviews` module also be supported?

Reply:
Added `countryviews` module to the list